src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx

```diff
@@ -12,7 +12,8 @@ export function GovernanceDAODashboard() {
 
   const accountTokenBalancesQueryResults = useAccountTokenBalances("");
   const { data: tokenBalances } = accountTokenBalancesQueryResults;
-
+  const { data: daoGovTokenBalance = 0 } = useTokenBalance({ tokenId: dao?.tokenId ?? "" });
```

Comment:
In GOV DAO an admin can have 0 or more than zero tokens in that case what will be the condition for the admin.?
- Is it necessary for the Admin to have GOV DAO Token if yes should it be more than zero to show user as admin.?

Reply:
I'm not 100% sure - please forward this question to Raj. My guess is that "Admin" tag should show up if the connected wallet matches the `adminId` in the solidity events regardless if they have gov tokens in their wallet. I would also ask Raj and the SC team the exact list of expected and existing privileges an admin has for the Governance and NFT DAOs. That may inform how we handle the UX for the Member/Admin tag.

Reply:
Sure, for now I have kept it like if you admin no matter if you have GOV token or not Admin Indicator will come up.